.travis.yml

```diff
@@ -4,22 +4,25 @@ node_js:
   - 6
   - 8
   - 9
+env:
+  global:
+    - PYTHON_VERSIONS="2.7.11 3.6.3"
+  matrix:
```

Comment:
this runs a separate build for each version of python under test, rather than a single build that tests both pythons